terra-jupyter-base/Dockerfile

```diff
@@ -26,6 +26,9 @@ RUN rm /etc/apt/sources.list.d/cuda.list \
     && curl -L -O https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-keyring_1.0-1_all.deb \
     && sudo dpkg -i cuda-keyring_1.0-1_all.deb
 
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+RUN sudo curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
```

Comment:
the sudo on curl should be unnecessary (curl itself doesn't need root privileges, just apt-key)

Reply:
I THINK we're already root as well, so I'm not sure we even need any `sudo`s 😆